src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -908,8 +908,8 @@ window.qBittorrent.DynamicTable ??= (() => {
             if (this.dynamicTableDiv.offsetHeight === 0)
                 return;
             this.renderedHeight = this.dynamicTableDiv.offsetHeight;
-            // show extra 6 rows at top/bottom to reduce flickering
-            const extraRowCount = 6;
+            // show extra rows at top/bottom to reduce flickering
+            const extraRowCount = 20;
```

Comment:
Would it be better to make it depends on something rather than some fixed value?
For example, a fraction (`0.5`) of the table height or `0.3` of the screen size.

Reply:
I don't think those dynamic values would impact how quickly the user scrolls, so there wouldn't be any benefit compared to using a static value.